defender/staged/downloaded/Forta_Large_Borrows_Governance.spec.js

```diff
@@ -0,0 +1,123 @@
+  governanceLevel: 'proposal',
+  minCOMPNeeded: '11',
+  currCOMPOwned: '12',
+};
+
+const {
+  Finding, FindingType, FindingSeverity,
+} = require('forta-agent');
+
+const mockFinding = Finding.fromObject({
+  name: 'Placeholder Alert',
+  description: 'Placeholder description',
+  alertId: 'AE-ALERT-ID',
+  type: FindingType.Info,
+  severity: FindingSeverity.Info,
+  protocol: 'Protocol',
+  metadata: mockMetadata,
+});
+
+// grab the existing keys before loading new content from the .env file
+const existingKeys = Object.keys(process.env);
+// eslint-disable-next-line import/no-unresolved
+require('dotenv').config();
+
+// now filter out all of the existing keys from what is currently in the process.env Object
+const newKeys = Object.keys(process.env).filter((key) => existingKeys.indexOf(key) === -1);
+const secrets = {};
+newKeys.forEach((key) => {
+  secrets[key] = process.env[key];
+});
+
+// Map the Env name to the Secret variable name
+if (discordSecretName !== discordEnvSecretName) {
+  secrets[discordSecretName] = secrets[discordEnvSecretName];
+  delete secrets[discordEnvSecretName];
+}
+
+// eslint-disable-next-line import/no-useless-path-segments
+const { handler } = require('../downloaded/Forta_Large_Borrows_Governance');
+
+function createFortaSentinelEvent(finding, blockHash, txHash) {
+  // Generally findings go from the Bot, to Scan Node, to Sentinel, to Autotasks
+  //  with some metadata being added and removed along the way. This function will mimic
+  // the Sentinel output with only Finding, block and transaction data.
+
+  // Note: Much of the extra data here is superfluous but is left here just in case future bots
+  // want to reference any of the Sentinel data in the Discord output. It also mimics sentinel
+  // output more accurately.
+
+  // populate the matchReasons Array with placeholders
+  const matchReasons = [
+    {
+      type: 'alert-id',
+      value: 'ALERT_ID_PLACEHOLDER',
+    },
+    {
+      type: 'severity',
+      value: 'INFO',
+    },
+  ];
+
+  // populate the sentinel Object with placeholders
+  const sentinel = {
+    id: '8fe3d50b-9b52-44ff-b3fd-a304c66e1e56',
+    name: 'Sentinel Name Placeholder',
+    addresses: [],
+    agents: [],
+    network: 'mainnet',
+    chainId: 1,
+  };
+
+  const autotaskEvent = {
+    relayerARN: undefined,
+    kvstoreARN: undefined,
+    credentials: undefined,
+    tenantId: undefined,
+    secrets,
+    request: {
+      body: {
+        hash: '0xAGENT-HASH', // forta Agent hash
+        alert: {
+          metadata: finding.metadata,
+        },
+        source: {
+          transactionHash: txHash,
+          block: {
+            hash: blockHash,
+          },
+        },
+        matchReasons,
+        sentinel,
+        type: 'FORTA',
+      },
+    },
+  };
+  return autotaskEvent;
+}
+
+describe('check autotask', () => {
+  it('Runs autotask against mock data and posts in Discord (manual-check)', async () => {
```

Comment:
Use the approach that was put in place from AE-1313 to mock `axios` and capture any outgoing requests to perform an `expect()` on.